packages/e2e-tests/specs/block-transforms.test.js

```diff
@@ -159,7 +159,11 @@ maybeDescribe( 'Block transforms', () => {
 			)
 		);
 
-		it.each( testTable )(
+		// Filter out Group transforms for perf reasons, expect for x2 tests
```

Comment:
There's little context for me here reviewing this pull request, let alone for the future maintainer. I don't know what "performance reasons" this refers to.

Reply:
You are right to flag this. I was explaining implementation details rather than outlining the real reason why we needed to filter out the Group transforms. I've now updated the comment to provide what I hope is a better explanation.